frontend/src/libs/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -35,12 +35,17 @@ const DropdownMenu: React.FC<Properties> = ({ routes }) => {
         )}
       >
         {routes.map((item) => {
+          const isRootPathSelected = pathname === item.path;
+          const isOtherPathSelected =
```

Comment:
Create a common helper for this and use enum as value instead of '/'

Reply:
Yes, changed it!